### DISCUSSION TITLE
Mac OS 10.0 issue

### DISCUSSION THREAD

**Discussion by Gamerboo0fficial** [utmapp/UTM#5074] [UTM Help (macOS)]
when getting a Mac OS 10.0 iso and booting it into UTM my trackpad nor any key work i will attach a screenshot.  https://imgur.com/a/vZx9KjD.  please help me asap

**Comment by ideologysec** [comment]
Please provide additional details. What macOS version is your host? Apple Silicon or Intel on your host hardware?

Please provide a config.plist for your VM and any debug logs that you can.

**Comment by Gamerboo0fficial** [comment]
i am using Mac os 12.1 monterey intel mac

also where can i find the config.plist file if i cant find it in time

**Comment by Gamerboo0fficial** [comment]
im attachig a video 

> i hate how this wont work

https://www.youtube.com/watch?v=OH8fNJryklE


pls help

**Comment by ideologysec** [comment]
Start here:

https://blog.greggant.com/posts/2021/01/13/install-powerpc-macos-osx-on-apple-silicon-m1-and-x86-intel.html

**Comment by Gamerboo0fficial** [comment]
i tryed it and it doesnt work dispite me having admin it says /user/local/cellular isnt writable 

is it possible to change the install directory?

**Comment by ideologysec** [comment]
You don't need to install qemu via Homebrew. Use the provided options and pass them as custom qemu options to UTM.

You'll need to boot an installer that can format disks though.

**Comment by barretogielson05-code** [comment]
ID: 15167716530652

ID 2: 84167715410050
